### PR TITLE
[SPARK-5093] Set spark.network.timeout to 120s consistently.

### PR DESCRIPTION
**Pull request by rxin**

### CODE REVIEW

**Comment by aarondav** [comment]
LGTM

**Comment by rxin** [comment]
Thanks. Merging in master.

File: core/src/main/scala/org/apache/spark/storage/BlockManagerMasterActor.scala [hunk 1]
```diff
@@ -52,11 +52,7 @@ class BlockManagerMasterActor(val isLocal: Boolean, conf: SparkConf, listenerBus
 
   private val akkaTimeout = AkkaUtils.askTimeout(conf)
 
-  val slaveTimeout = {
-    val defaultMs = math.max(conf.getInt("spark.executor.heartbeatInterval", 10000) * 3, 45000)
-    val networkTimeout = conf.getInt("spark.network.timeout", defaultMs / 1000)
-    conf.getLong("spark.storage.blockManagerSlaveTimeoutMs", networkTimeout * 1000)
-  }
+  val slaveTimeout = conf.getLong("spark.storage.blockManagerSlaveTimeoutMs", 120 * 1000)
```
Comment (lianhuiwang) [inline core/src/main/scala/org/apache/spark/storage/BlockManagerMasterActor.scala]: if user do not set blockManagerSlaveTimeoutMs, i think we need to set spark.network.timeout to blockManagerSlaveTimeoutMs. if spark.network.timeout is not defined, we can set the default 120 to spark.network.timeout.

Comment (rxin) [inline core/src/main/scala/org/apache/spark/storage/BlockManagerMasterActor.scala]: I think it's fair to remove the blockManagerSlaveTimeoutMs. The various timeouts are not useful ... Let me check with others also.

Comment (rxin) [inline core/src/main/scala/org/apache/spark/storage/BlockManagerMasterActor.scala]: cc @pwendell and @tdas (I think you wrote this config?)

**Pull request closed without merging**